src/Nut/TextConverters/BaseConverter.cs

```diff
@@ -1,212 +1,215 @@
+      {
+        builder.AppendFormat("{0} ", NumberTexts[num][0]);
+      }
+    }
+
+    protected virtual void AppendUnitsForAdditional(long num, StringBuilder builder)
+    {
+      if (num > 0)
+      {
+        builder.AppendFormat("{0} ", NumberTexts[num][0]);
+      }
+    }
+
+    protected virtual long AppendTens(long num, StringBuilder builder)
+    {
+      if (num > 20)
+      {
+        var tens = ((int)(num / 10)) * 10;
+        builder.AppendFormat("{0} ", NumberTexts[tens][0]);
+        num = num - tens;
+      }
+      return num;
+    }
+
+    protected virtual long AppendHundreds(long num, StringBuilder builder)
+    {
+      if (num > 99)
+      {
+        var hundreds = num / 100;
+        builder.AppendFormat("{0} {1} ", NumberTexts[hundreds][0], NumberTexts[100][0]);
+        num = num - (hundreds * 100);
+      }
+      return num;
+    }
+
+    private static void NumberLimitControl(long num)
+    {
+      if (num >= Parameters.NumberLimit)
+      {
+        throw new Exception(string.Format("{0} and larger than {0} numbers are not supported", Parameters.NumberLimit));
+      }
+    }
+
+    #region Currency
+
+    public virtual string ToText(decimal num, string currency, Options options)
+    {
+      return ToText(num, currency, options, GenderGroup.None);
+    }
+
+    public virtual string ToText(decimal num, string currency, Options options, GenderGroup genderGroup = GenderGroup.None)
+    {
+      var builder = new StringBuilder();
+      if (currency == Currency.TL) currency = Currency.TRY;
+      var currencyModel = GetCurrencyModel(currency);
+      if (currencyModel == null) return string.Empty;
+      var decimalSeperator = num.ToString(CultureInfo.InvariantCulture).Contains(",") ? ',' : '.';
+      var nums = num.ToString(CultureInfo.InvariantCulture).Split(decimalSeperator).ToList();
+      if (nums.Count == 1) nums.Add("00");
+
+      var mainUnitNum = Convert.ToInt64(nums[0]);
+
+      if (options.MainUnitNotConvertedToText)
+      {
+        builder.Append(nums[0]);
+      }
+      else
+      {
+        var mainUnitText = ToText(mainUnitNum, currencyModel, true);
+        mainUnitText = options.MainUnitFirstCharUpper ? mainUnitText.ToFirstLetterUpper(CultureName) : mainUnitText;
+        builder.Append(mainUnitText);
+      }
+
+      builder.Append(" ");
+
+      var currencyText = GetCurrencyText(mainUnitNum, currencyModel);
+      currencyText = options.CurrencyFirstCharUpper ? currencyText.ToFirstLetterUpper(CultureName) : currencyText;
+      builder.Append(currencyText);
+
+
+      if (nums.Count > 1 && !string.IsNullOrEmpty(nums[1]))
+      {
+        var subUnitText = nums[1].PadRight(2, '0');
+        var subUnitNum = Convert.ToInt64(subUnitText);
+        if (!options.SubUnitZeroNotDisplayed || subUnitNum != 0)
+        {
+          if (options.RadixInCurrencyNotConvertedToText)
+            builder.Append(" ");
+          else
+            builder.Append(GetUnitSeparator(currencyModel));
```

Comment:
Strange behaviour of git, I only change the line `builder.Append(GetUnitSeparator(currencyModel));` with this four lines